### PR TITLE
Updating dropdown topic

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/components/dropdown.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
 
 # Dropdown
 
-Use the Dropdown Component to let the user select an item from a collection that is displayed upon user interaction in a scrollable list. Only one item can be selected at a time and if your scenario allows for multiple selection, you should use the [Combo](combo.md). The Dropdown is visually identical to the [Ignite UI for Angular Dropdown Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop_down.html)
+Use the Dropdown Component to let the user select an item from a collection that is displayed upon user interaction in a scrollable list. Only one item can be selected at a time and if your scenario allows for multiple selections, you should use the [Combo](combo.md). The Dropdown is visually identical to the [Ignite UI for Angular Dropdown Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop_down.html)
```
Comment (StefanIvanov) [inline en/components/dropdown.md:9]:
```suggestion
Use the Dropdown Component to let the user select an item from a collection that is displayed upon user interaction in a scrollable list. Only one item can be selected at a time and if your scenario requires the selection of multiple items, you should use the [Combo](combo.md) instead. The Dropdown is visually identical to the [Ignite UI for Angular Dropdown Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop_down.html)
```

File: en/components/dropdown.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ The Dropdown supports two types of items header and item. Through headers it is
 
 ## Item States
 
-The Dropdown items support five different states: disabled, **idle**, focused, selected and focused&selected.
+The Dropdown items support five different states: disabled, **idle**, focused, selected and selected&focused.
```
Comment (StefanIvanov) [inline en/components/dropdown.md:31]:
```suggestion
The Dropdown items support five different states: Disabled, **Idle**, Focused, Selected and Selected&focused.
```

File: en/components/dropdown.md [hunk 3]
```diff
@@ -44,17 +44,12 @@ The Dropdown comes with styling flexibility through an override controlling the
 
 ## Usage
 
-When using the Dropdown you should always show it on top of the content that follows the item which triggers its display. Dopdowns don't push content like accordions do.
+When using the Dropdown you should always show it on top of the content that follows the item which triggers its display. Dropdowns don't push content like accordions do.
```
Comment (StefanIvanov) [inline en/components/dropdown.md:47]:
```suggestion
When using the Dropdown you should always show it on top of the content that follows the item which triggers its display. Dropdowns don't push content like an expansion panel would.
```

File: en/components/dropdown.md [hunk 4]
```diff
@@ -28,14 +28,20 @@ The Dropdown supports two types of items header and item. Through headers it is
...
 <img class="responsive-img" src="../images/dropdown_item_idle.png" srcset="../images/dropdown_item_idle@2x.png 2x" />
 <img class="responsive-img" src="../images/dropdown_item_focused.png" srcset="../images/dropdown_item_focused@2x.png 2x" />
 <img class="responsive-img" src="../images/dropdown_item_selected.png" srcset="../images/dropdown_item_selected@2x.png 2x" />
 <img class="responsive-img" src="../images/dropdown_item_selected_focused.png" srcset="../images/dropdown_item_selected_focused@2x.png 2x" />
 
+## Item Layout Template
+
+The Dropdown Item supports flexible icon and label templating achievable by setting the elements you want to hide to ~No Symbol from the overrides panel. By default the icon is hidden with this approach.
+
+<img class="responsive-img" src="../images/dropdown_item_idle.png" srcset="../images/dropdown_item_idle@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/components/dropdown.md:43]: add a second image with both icon and label

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/dropdown.md (hunk 4 above)
Comment (StefanIvanov) [reply en/components/dropdown.md:43]: only adding a second image here seems to be left and I will approve the changes

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**